app/models/node-request.ts

```diff
@@ -0,0 +1,19 @@
+// app/models/node-request.js
+import Model, { attr, belongsTo } from '@ember-data/model';
+
+export enum RequestTypeChoices {
+    AccessRequest = 'access',
+    InstitutionalRequest = 'institutional_request',
+}
+
+export default class NodeRequestModel extends Model {
+  @attr('string') comment;
+  @attr('string') requestedPermissions;
+  @attr('string') requestType;
+  @attr('boolean') bccSender;
+  @attr('boolean') replyTo;
+  @belongsTo('institution') institution;
+  @belongsTo('user') messageRecipent;
+  @belongsTo('node') node;
```

Comment:
For these `belongsTo` relationships, we typically manually type them like
```suggestion
  @belongsTo('institution')
  institution!: AsyncBelongsTo<InstitutionModel> & InstitutionModel;
```
and so on with `import { AsyncBelongsTo } from '@ember-data/model';`